hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -8785,9 +8785,13 @@ public void logAuditEvent(boolean succeeded, String userName,
         src = escapeJava(src);
         dst = escapeJava(dst);
         sb.setLength(0);
+        // InetAddress#tostring can also return hostname in addition to IP address.
+        // To not include hostname regardless of the hostname resolution, we should
+        // only include IP address here.
```

Comment:
I have one more request. Given that the name of `ipAddr` has a hint of the comment, we may be able to delete the comment.

Reply:
Done